src/components/ExpandableSection/__tests__/ExpandableSection.spec.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { Button } from '../../../';
+import ExpandableSection from '../ExpandableSection';
+
+const getExpandableSection = (props = {}) => shallow(<ExpandableSection id="51er" {...props} />);
+
+
+describe('<ExpandableSection />', () => {
+  it('renders right class name', () => {
+    const mounted = getExpandableSection();
```

Comment:
`const mounted = getExpandableSection({ open: true });` no need to use setProps here if you use a fn to get the component